frontend/src/metabase/components/AppBanner/AppBanner.tsx

```diff
@@ -66,6 +66,10 @@ export const AppBanner = () => {
     return <PaymentBanner tokenStatus={tokenStatus} />;
   }
 
+  if (isDevMode) {
```

Comment:
nit: I'd probably just put this if statement above the `!isAdmin` condition

Reply:
That's fair, although my reasoning was that the order the banners appear in code matter, because the conditions are not mutually exclusive. For example if a trial is active and dev mode is also present, then only one banner gets shown. Dev mode is the least important banner, which is why I added it last.

However non-admins still need to see it and it sort of makes sense to short circuit early for non-admins